.eslintrc.js

```diff
@@ -17,6 +17,8 @@ module.exports = {
     'node_modules/**/*',
     'jest-coverage/**/*',
   ],
+  // This is necessary to run eslint on Windows and not get a thousand CRLF errors
+  rules: { 'prettier/prettier': ['error', { endOfLine: 'auto' }] },
```

Comment:
can we separate this out to a different PR? Where we can have a separate discussion about this?